internal/service/privatelinkendpoint/resource_privatelink_endpoint_test.go

```diff
@@ -188,7 +188,7 @@ func testAccCheckMongoDBAtlasPrivateLinkEndpointImportStateIDFunc(resourceName s
 	}
 }
 
-func testAccCheckMongoDBAtlasPrivateLinkEndpointExists(resourceName string) resource.TestCheckFunc {
+func checkExists(resourceName string) resource.TestCheckFunc {
```

Comment:
still using acc.Conn() instead of ConnV2

Reply:
ah! I got fooled because I looked at the imports and there was no import for the client, my bad. fixing it